scripts/cocktailsToMakeModel.js

```diff
@@ -0,0 +1,54 @@
+var cocktailsToMake = {
+  ingredientString: '',
+  paramString: '',
+  cocktailObjects: [],
+  listTemplate: '',
+
+  createIngredientString: function (){
+    $('#addedIngredients li').each(function(index){
+      cocktailsToMake.ingredientString += ('"' + $(this).text() + '"' + ', ');
+    });
+    var lastIndex = cocktailsToMake.ingredientString.lastIndexOf(',');
+    return cocktailsToMake.ingredientString.substr(0, lastIndex);
+  },
+
+  createParamString: function (rows){
+    rows.forEach(function (row){
+      cocktailsToMake.paramString += (row.ID + '-');
+    });
+    var lastIndex = cocktailsToMake.paramString.lastIndexOf('-');
```

Comment:
if you iterate through an array of objects that has length of more than 1 and append a dash at the end of it, you know the index of the last dash. It's the length of the string minus 1.